packages/gasket-core/lib/gasket.js

```diff
@@ -58,11 +58,11 @@ export class Gasket {
     // Can be used as a key to identify a gasket instance
     this.symbol = Symbol('gasket');
 
-    // @ts-ignore
+    // @ts-ignore - attached lifecycle trace methods not recognized
```

Comment:
You could add an `implements GasketEngine` or something like that on the class if you have things like `exec` in an interface.

Reply:
So, we already "extend" `GasketEngine` [here](https://github.com/godaddy/gasket/blob/edge-runtime/packages/gasket-core/lib/index.d.ts#L113). It works in consuming code.

Reply:
We took a look but didn't find a working option. I created a ticket to cycle back for an audit of these ts-ignores.